src/thunderbots/software/util/ros_messages.h

```diff
@@ -96,5 +96,45 @@ namespace Util
             const thunderbots_msgs::RefboxCommand& command);
 
         World createWorldFromROSMessage(const thunderbots_msgs::World& world_msg);
+
+        /**
+         * Given a World message, it will transform the world's orientation
```

Comment:
how will it transform the orientation? Same goes for the other javadoc comments here.